handler/handler.go

```diff
@@ -57,6 +60,11 @@ func (batchHandler BatchHandler) sendToES(ctx context.Context, esDestURL string,
 
 	var bulkupsert []byte
 	for _, event := range events {
+		if event.UID == "" {
```

Comment:
Is this the only validation available?

Reply:
validations will be address in separate tasks, no current requirements